docs/03-tutorials/04-design-automation/_shared/dotnet-vs2022/setup/config.mdx

```diff
@@ -1,14 +1,14 @@
-From the solution explorer, select `designAutomationsample` project, under the `Properties`,
+From the solution explorer, select `designAutomationSample` project, under the `Properties`,
 select and open the `launchSettings.json`
 
 - Change the `applicationUrl` to "http://localhost:8080"
-- Go to profiles\forgeSample\environmentVariables add following environments.
+- Inside the `launchSettings.json` go to `profiles\designAutomationSample\environmentVariables` add following environments.
```

Comment:
Not a big thing, but should this say _add following environment variables_ instead of _add following environments_?